modelon/impact/client/entities/experiment.py

```diff
@@ -374,6 +430,37 @@ def get_trajectories(self, variables: List[str]) -> Dict[str, Any]:
             for j in case_nbrs
         }
 
+    def get_last_time_point(self, variables: List[str]) -> ExperimentResultPoint:
+        """Returns a ExperimentResultPoint class for a list of result variables
+        for all the cases.
+
+        Args:
+            variables: A list of result variables to fecth trajectories for.
+
+        Returns:
+            An ExperimentResultPoint class object.
+
+        Raises:
+            OperationNotCompleteError if simulation process is in progress.
+            OperationFailureError if simulation process was cancelled.
+            TypeError if the variable is not a list object.
+            ValueError if trajectory variable is not present in the result.
+
+        Example::
+
+            result = experiment.get_last_time_point(['h', 'time'])
+
+            # Convert to Pandas data frame
+            df = pd.DataFrame(data=result.as_lists(), columns=result.variables,
+                index=result.cases)
+            df.index.name = "Cases"
+
+        """
+        trajectories = self._validate_and_fetch_trajectories(
+            variables, True, _TrajectoryResponseFormat.v2
```

Comment:
I suggest using keyword arguments, e.g.,  `only_last_point=True`, to enhance readability.